types/youtube-player.ts

```diff
@@ -0,0 +1,189 @@
+// TODO: fully type definitions for youtube-player
```

Comment:
Feel free to use my partial typings.

https://github.com/ArjixWasTaken/youtube-music-ts/blob/master/src/shared/types/PlayerAPI.ts

Reply:
The changes are so many that the GitHub UI is struggling when viewing them 🤣